exercises/bracket-push/test/bracket_push_tests.erl

```diff
@@ -0,0 +1,52 @@
+-module(bracket_push_tests).
+
```

Comment:
Are these tests based on the canonical data? If so, could you please specify the exact version of the specification in a comment like this?

```erl
% based on canonical data version 0.0.0
% https://link to raw file data
```